src/app/introduction/picture-puzzle.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+    <div class="col-lg-8 col-lg-offset-3">
+        <h2>Take a picture of yourself, someone or something important to you</h2>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-3 col-md-offset-3">
+        <video #video width="480" height="360" autoplay [hidden]='webcamState!=webcamStates.OPENED'></video>
+        <canvas width="480" height="360" [hidden]='webcamState===webcamStates.OPENED' id="canvas"></canvas>
+    </div>
+    <div class="col-md-3 col-md-offset-3">
+        <input type="file" name="file" id="file" accept="image/*" (change)="handleInputChange($event)">
+        <button class="btn btn-block btn-danger btn-lg" *ngIf='webcamState===webcamStates.PAGE_LOAD' (click)="openWebcam()">{{webcamButtonText}}</button>
```

Comment:
as far as i can tell, there are only 3 states to `webcamStates`, names `PAGE_LOAD`, `CAPTURED`, & `OPENED`.
then, why not `*ngIf='webcamState===webcamStates.OPENED'` & `*ngIf='webcamState!==webcamStates.OPENED'` instead?